Css/Sections/Chatbot/chatbot.css

```diff
@@ -1,33 +1,106 @@
 /* Chatbot Styles*/
+.chat-toggle {
+    background-color:  var(--clr-primary-dark);
+    position: fixed;
+    right: 40px;
+    bottom: 20px;
+    height: 50px;
+    width: 50px;
+    border: none;
+    outline: none;
+    border-radius: 50%;
+    cursor: pointer;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    color: var(--clr-default);
+    z-index: 10;
+}
+.chat-toggle i{
+    position: absolute;
+    font-size: 1.8rem;
+}
+.show-chatbot .chat-toggle i:first-child,
+.chat-toggle i:last-child {
+    opacity: 0;
+}
+.show-chatbot .chat-toggle i:last-child {
+    opacity: 1;
+}
 .chatbot {
     background-color:  var(--clr-default);
     position: fixed;
-    z-index: 1;
+    z-index: 10;
     right: 40px;
     bottom: 100px;
-    width: 300px;
+    width: 100%;
+    max-width: 300px;
+    transform: scale(0.5);
+    opacity: 0;
+    pointer-events: none;
     overflow: hidden;
     border-radius: var(--brd-radius);
     box-shadow:  0 0 128px 0 rgba(0, 0, 0, 0.1), 0 32px 64px -48px rgba(0, 0, 0, 0.5);
 }
+.show-chatbot .chatbot {
+    transform: scale(1);
+    opacity: 1;
+    pointer-events: auto;
+}
 
 .chatbot header {
```

Comment:
create a white bottom border in the header to create some seperation between the header and the chat area